CODE/VulnScan/v2-deprecated/_train.py

```
@@ -0,0 +1,429 @@
+            logits = outputs.logits
+            predictions.extend(torch.argmax(logits, dim=1).cpu().numpy())
+
+    logging.info("BERT model evaluating..")
+    evaluate_model(y_test, np.array(predictions))
+    model.save_pretrained(os.path.join(SAVE_DIR, "bert_model"))
+    logging.info("Model saved as bert_model")
+
+
+class LSTMModel(nn.Module):
+    def __init__(self, vocab_size: int, embedding_dim: int = 128, hidden_dim: int = 128, output_dim: int = 1):
+        super(LSTMModel, self).__init__()
```

Ruff (0.8.0)

Use `super()` instead of `super(__class__, self)`

Remove `__super__` parameters

(UP008)